aim/tests/unit/test_aim_manager.py

```diff
@@ -1356,11 +1360,12 @@ class TestSecurityGroupRuleMixin(object):
                                 'security_group_subject_name': 'subject1',
                                 'name': 'rule1',
                                 'direction': 'ingress',
-                                'remote_ips': ['10.0.0.1/30',
```

Comment:
By removing these remote IPs, do we lose some coverage for a UT?

Reply:
It shouldn't because I added those back in the test_update_attributes.